egs/sprakbanken_swe/s5/local/sprak_data_prep.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+# Copyright 2009-2012  Microsoft Corporation  Johns Hopkins University (Author: Daniel Povey)
+# Copyright 2013-2014  Mirsk Digital Aps (Author: Andreas Kirkedal)
+#Copyright 2016 KTH Royal Institute of Technology (Author: Emelie Kullmann)
+# Apache 2.0.
+
+
+dir=`pwd`/data/local/data
+lmdir=`pwd`/data/local/transcript_lm
+traindir=`pwd`/data/local/trainsrc
+testdir=`pwd`/data/local/testsrc
+rm -rf $lmdir $traindir $testdir $devdir
+mkdir -p $dir $lmdir $traindir $testdir $devdir
+local=`pwd`/local
+utils=`pwd`/utils
+
+
+. ./path.sh
+
+# Checks if python3 is available on the system and install python3 in userspace if not
+# This recipe currently relies on version 3 because python3 uses utf8 as internal 
+# string representation
+
+if ! which python3 >&/dev/null; then
+  echo "Installing python3 since not on your path."
+  pushd $KALDI_ROOT/tools || exit 1;
+  extras/install_python3.sh || exit 1;
+  popd
+fi
+
```

Comment:
I'm not particularly happy about python3
a) could you elaborate why you need the internal encoding of the strings to be utf-8? Given the fact you are using codecs with (what I'm assuming) correct encoding, the internal representation will be always correct.
b) the install script is not provided -- and perhaps it would be better just to ask user to install system package (or run the script manually). 
c) the python scripts have hashbang `/usr/bin/env python`, not `/usr/bin/env python3` -- I noticed you call some scripts manually (`python3 name-of-the-script.py`) but I think these things should be in sync (if you are absolutely sure you need python3)
d) I think the current strategy in kaldi is to have python scripts working in python 2.7 (but compliant with python3)